packages/web-react/src/components/Tooltip/__tests__/TooltipPopover.test.tsx

```diff
@@ -17,7 +17,7 @@ describe('TooltipPopover', () => {
     const popoverText = 'TooltipPopover';
 
     const dom = render(<TooltipPopover>{popoverText}</TooltipPopover>);
-    const element = dom.container.querySelector('.Tooltip') as HTMLElement;
+    const element = dom.container.querySelector('.TooltipPopover') as HTMLElement;
```

Comment:
Nice-to-have: refactor this test according to the new approach. :-)

Reply:
refactored 👍